processes/security_team_membership_policy.md

```diff
@@ -0,0 +1,27 @@
+# Node.js Security Team Membership Policy
+
+The Node.js Security Team (@nodejs/security) has
+access to security-sensitive issues and patches that aren't appropriate for
+public availability.
+
+> Note: Due to implementation details and a bit of human error, the list of
+> people with access to the security patches isn't 100% in line with the
+> @nodejs/security membership. See the [membership list](./security_team_members.md)
+> for details. This situation is intended to be corrected in the near future.
+
+This is different than the security triage team, which is defined by the
+[`security@nodejs.org` email alias](https://github.com/nodejs/email/blob/master/iojs.org/aliases.json).
+Inclusion in that group is on a volunteer basis, upon approval by the Technical
+Steering Committee (TSC).
+
+The policy for inclusion on the @nodejs/security is as follows:
+
+1. All members of @nodejs/TSC are members of @nodejs/security.
+2. On a case-by-case basis, individuals outside the Technical Steering Committee
+   are invited by the TSC to join @nodejs/security so that their expertise can
+   be applied to an issue or patch. Once their assistance is no longer required
+   for a particular issue or patch, they are removed from the team.
+3. Members of the [release team](https://github.com/nodejs/node#release-team)
+   are also members of @nodejs/security, as they need access to the patches in
+   order to produce releases.
+4. Members of the security triage team are members of @nodejs/security.
```

Comment:
This should also specify how someone becomes a member of he triage team ... specifically, the triage team should be a volunteer subset of @nodejs/security approved by the larger team.

Reply:
I think triage is currently approved by TSC, not the security team as a whole.